en/developer_guide/specific_backbone_view.rst

```diff
@@ -0,0 +1,40 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+As all the Backoffice of Open Orchestra is based on Backbone view system, all the specific event
+linked to a view are linked to the view directly.
+
+In some cases, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page will not suit your needs. There are some new behaviour to add to the form, such as a
+specific event when you edit a field.
```

Comment:
I find the general explanation better than the first version, but I'll add a little explanation on how an edit form page is built: two templates are involved. The first is the backbone view for the 'container' aspect (including events binding), the second is the template form describing the rendering of the fields.
The good practice you're proposing is to bind the events on the backbone view rather than the form template.